src/rdiffbackup/locations/repository.py

```diff
@@ -86,6 +92,9 @@ def check(self):
         if self.must_be_writable and not self._is_writable():
             ret_code |= 1
 
+        if self._is_locked():
```

Comment:
If I'm reading this the right way, this might return True in case a previous backup failed and leave the ".lock" file.

Reply:
Correct. Unless you have the force option.

Reply:
New version does check if the process which created the lockfile is still running with the same PID _and_ name.